pkg/cmd/eventlistener/describe.go

```diff
@@ -0,0 +1,198 @@
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package eventlistener
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"text/tabwriter"
+	"text/template"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/formatted"
+	"github.com/tektoncd/cli/pkg/printer"
+	"github.com/tektoncd/cli/pkg/validate"
+	"github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	cliopts "k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+const describeTemplate = `{{decorate "bold" "Name"}}:	{{ .EventListener.Name }}
+{{decorate "bold" "Namespace"}}:	{{ .EventListener.Namespace }}
+
+{{- if ne .EventListener.Spec.ServiceAccountName "" }}
+{{decorate "bold" "Service Account"}}:	{{ .EventListener.Spec.ServiceAccountName }}
+{{- end }}
+
+{{- if ne .EventListener.Spec.ServiceType "" }}
+{{decorate "bold" "Service Type"}}:	{{ .EventListener.Spec.ServiceType }}
+{{- end }}
+
+{{- $value := getEventListenerTrigger .EventListener }}
+{{-  if ne $value "" }}
+
+{{decorate "resources" ""}}{{decorate "underline bold" "EventListenerTrigger\n"}}
+{{ $value }}
+{{- end }}
+
+{{- $value := getURL .EventListener }}
+{{-  if ne $value "" }}
+{{decorate "green" "URL"}}:{{ $value }}
```

Comment:
There should be a space here:

```suggestion
{{decorate "green" "URL"}}: {{ $value }}
```